src/App.tsx

```diff
@@ -1,19 +1,173 @@
 import React from 'react';
-import './App.css';
+import { TodoList } from './components/TodoList';
+
+import { Todo } from './types/Todo';
+
+import './App.scss';
 
 import users from './api/users';
+import todos from './api/todos';
+
+const maxId = todos.sort((a, b) => b.id - a.id)[0].id;
+
+type Props = {};
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+type State = {
+  newTodos: Todo[],
+  title: string;
+  userId: number;
+  id: number;
+  titleWarning: boolean;
+  userWarning: boolean;
 };
 
-export default App;
+export class App extends React.Component<Props, State> {
+  state = {
+    newTodos: [...todos],
+    title: '',
+    userId: 0,
+    id: maxId,
+    titleWarning: false,
+    userWarning: false,
+  };
+
+  todoMaker = () => {
+    const {
+      title,
+      userId,
+    } = this.state;
+
+    this.nextId();
+
+    if (!title) {
+      this.setState({ titleWarning: true });
+    }
+
+    if (userId === 0) {
+      this.setState({ userWarning: true });
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    (title && userId !== 0) && (
+      this.setState((state) => ({
+        newTodos: [
+          ...state.newTodos,
+          {
+            userId: +state.userId,
+            id: state.id,
+            title: state.title,
+            completed: false,
+          },
+        ],
+        title: '',
+        userId: 0,
+      }))
+    );
+  };
+
+  nextId = () => {
+    this.setState(current => ({
+      id: current.id + 1,
+    }));
+  };
+
+  render() {
+    const {
+      title,
+      userId,
+      newTodos,
+      titleWarning,
+      userWarning,
+    } = this.state;
+
+    return (
+      <div className="App">
+        <div className="warnings App__warnings">
+          {titleWarning && (
+            <div className="warnings__warning warnings__warning--title">
+              Please write a title
+            </div>
+          )}
+          {userWarning && (
+            <div className="warnings__warning warnings__warning--user">
+              Please choose a usere
+            </div>
+          )}
+        </div>
+        <div className="content App__content">
+          <h2 className="title content__title">
+            Add todo form
+          </h2>
+          <form
+            action="/"
+            method="get"
+            className="form content__form"
+            onSubmit={(event) => {
+              event.preventDefault();
+              this.todoMaker();
+            }}
```

Comment:
move event.preventDefault inside createTodo

```suggestion
            onSubmit={this.createTodo}
```